index.js

```diff
@@ -8,18 +8,88 @@ module.exports = {
+                    },
+                    'q-applicant-declaration': {
+                        title: 'Do you agree to this declaration?',
+                        type: 'array',
+                        items: {
+                            anyOf: [
+                                {
+                                    title: 'Yes',
+                                    const: true
+                                }
+                            ]
+                        }
                     }
                 },
-                examples: [{}],
+                errorMessage: {
+                    required: {
+                        'q-applicant-declaration': 'Select agree'
+                    }
+                },
+                examples: [
+                    {
+                        'q-applicant-declaration': [true]
+                    }
+                ],
                 invalidExamples: [
                     {
-                        foo: 'bar'
+                        'q-applicant-declaration': 'not-an-array'
+                    },
+                    {
+                        'q-applicant-declaration': ['not-a-key']
+                    }
+                ]
+            }
+        },
+        'p-main-applicant-declaration': {
+            schema: {
+                $schema: 'http://json-schema.org/draft-07/schema#',
+                type: 'object',
+                required: ['q-main-applicant-declaration'],
+                additionalProperties: false,
+                properties: {
+                    'main-applicant-declaration': {
+                        title: 'Declaration',
+                        description:
+                            'This is the blurb for a main applicant - with or without an applicant who is over or under 12 years of age.'
+                    },
+                    'q-main-applicant-declaration': {
+                        title: 'Do you agree to this declaration?',
+                        type: 'array',
+                        items: {
+                            anyOf: [
+                                {
+                                    title: 'Yes',
+                                    const: true
+                                }
+                            ]
+                        }
+                    }
```

Comment:
Same here, re: Compared to the prototype and schema validation.

Reply:
see other comment re prototype